src/data/part.lean

```diff
@@ -200,6 +200,14 @@ dif_neg id
   get_or_else (some a) d = a :=
 dif_pos trivial
 
+lemma get_or_else_of_dom (a : part α) (h : a.dom) [decidable a.dom] (d : α) :
+  get_or_else a d = a.get h :=
+begin
```

Comment:
Isn't `dif_pos h` enough to prove this?

Reply:
rewrite tactic failed, did not find instance of the pattern in the target expression
  dite ?m_2 ?m_4 ?m_5

Reply:
Perhaps you meant to use apply? That works, yes.